packages/terra-alert/src/_variables.scss

```diff
@@ -0,0 +1,71 @@
+$terra-alert-alert-color-scheme: (
```

Comment:
I believe I did this as part of the update to CSS modules. If you could look at this as part of that change, I would appreciate it.